pkg/data/task_partial_result.go

```diff
@@ -0,0 +1,14 @@
+package data
+
+const EventTypeTaskResultPartial EventType = "task_result_partial"
```

Comment:
You also need to add parsing of this event type on `parse.go`: https://github.com/livepeer/livepeer-data/blob/8ad1c8b2fbcac4f318c40b8545275521dfb69b85/pkg/data/parse.go#L44

(we don't really parse this specific event in Go today, but let's keep the lib complete for now)